packages/time-picker/src/TimePicker/components/MenuItem/types.ts

```diff
@@ -0,0 +1,34 @@
+export interface Node<T> {
```

Comment:
You mentioned this was copied from elsewhere right? We'd ideally not want to need to redefine this ourselves (especially not with the name `Node`). Let's try and find it to import :)